org.mwc.debrief.core/src/org/mwc/debrief/core/wizards/dynshapes/DynamicShapeBaseWizardPage.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * 
+ */
+package org.mwc.debrief.core.wizards.dynshapes;
+
+import java.util.regex.Pattern;
+
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Font;
+import org.eclipse.swt.graphics.FontData;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Label;
+
+/**
+ * @author Ayesha <ayesha.ma@gmail.com>
+ *
+ */
+public abstract class DynamicShapeBaseWizardPage extends WizardPage
+{
+  public static final String TIMINGS_PAGE = "Timings";
+  public static final String BOUNDS_PAGE = "Bounds";
+  public static final String STYLING_PAGE = "Styling";
+  private String pageName;

```

Comment:
@ayeshare - pageName can be made final.  We assign it once (in the constructor), then never modify it.

Reply:
@ayeshare - correction, we don't need the pageName field at all.  I see it gets passed to the parent, so we can retrieve it via `getName`;